pkg/innerring/innerring.go

```diff
@@ -828,19 +830,29 @@ func createClient(ctx context.Context, p *chainParams, notaryOpts ...client.Nota
 	)
 }
 
-func parseContracts(cfg *viper.Viper) (*contracts, error) {
+func parseContracts(cfg *viper.Viper, withOutMainNet bool) (*contracts, error) {
```

Comment:
without is a single word, not withOut :smiley:

Reply:
wow, a strange reflex of my fingers

fixed